Modules/OrgUnit/classes/class.ilObjOrgUnitTree.php

```diff
@@ -311,6 +311,33 @@ public function getEmployeesUnderUser($user_id, $recursive = true) {
 		return $employees;
 	}
 
+	// cat-tms-patch start (352)
+	/**
+	 * @param $user_id   int
+	 * @param $recursive bool if this is true subsequent orgunits of this users superior role get searched as well.
+	 * @return int[] returns an array of user_ids of the users which have an superior role in an orgunit of which this user's id has a superior role.
```

Comment:
@param int $user_id 
@param bool $recursive bool

Reply:
Versthe ich richtig?
This function returns all users with superior role in orgunits where $user_id is superior of.
If recursive is true, all superiors for any orgus below the ones $user_id is superior of will be returned?

Reply:
Verstehst du richtig.